conf/igenomes.config

```diff
@@ -36,7 +36,9 @@ params {
         'GATK.GRCh38' {
             ac_loci               = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/ASCAT/1000G_phase3_GRCh38_maf0.3.loci"
             ac_loci_gc            = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/ASCAT/1000G_phase3_GRCh38_maf0.3.loci.gc"
-            bwa                   = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/BWAIndex/Homo_sapiens_assembly38.fasta.64.{alt,amb,ann,bwt,pac,sa}"
+            bwa                   = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/BWAIndex/"
+            bwamem2               = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/BWAmem2Index/"
+            hashmap               = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/dragmap/"
```

Comment:
to be updated with proper value cc @FriederikeHanssen

Reply:
yep, it is on my agenda